File: scratch/check_assets.js
const { MongoClient } = require('mongodb');
const fs = require('fs');
const path = require('path');

// Read database url from .env
const envPath = path.join(__dirname, '..', '.env');
const envContent = fs.readFileSync(envPath, 'utf8');
const match = envContent.match(/DATABASE_URL=["']?([^"'\r\n]+)["']?/);
if (!match) {
  console.error("Could not find DATABASE_URL in .env");
  process.exit(1);
}
const uri = match[1];

async function main() {
  console.log("Connecting to:", uri.split('@').pop());
  const client = new MongoClient(uri);
  try {
    await client.connect();
    const db = client.db('rustapi');
    const col = db.collection('assets');
    const count = await col.countDocuments({});
    console.log("Total assets:", count);
    const docs = await col.find({}).limit(20).toArray();
    console.log("Sample documents:");
    console.log(JSON.stringify(docs, null, 2));
  } catch (err) {
    console.error(err);
  } finally {
    await client.close();
  }
}
main();

File: scratch/check_assets.py
import pymongo
import os
from dotenv import load_dotenv

load_dotenv()

uri = os.getenv("DATABASE_URL")
client = pymongo.MongoClient(uri)
db = client["rustapi"]
col = db["assets"]

print("Connecting to database:", uri.split("@")[-1])
total = col.count_documents({})
print("Total assets in DB:", total)

for a in col.find().limit(10):
    print({
        "id": str(a.get("_id")),
        "filename": a.get("filename"),
        "asset_type": a.get("asset_type"),
        "public_url": a.get("public_url")
    })
